core/src/main/java/org/projectnessie/cel/common/types/pb/FieldDescription.java

```diff
@@ -137,14 +138,14 @@ private static Class<?> reflectTypeOfField(FieldDescriptor fieldDesc) {
       case INT32:
       case SFIXED32:
       case SINT32:
-      case FIXED32:
         return Integer.class;
       case INT64:
       case SFIXED64:
       case SINT64:
         return Long.class;
       case UINT32:
       case UINT64:
+      case FIXED32:
```

Comment:
This change is to make reading fixed32 consistent. The other parts of the file considers fixed32s as a ULong (e.g. [here](https://github.com/projectnessie/cel-java/blob/main/core/src/main/java/org/projectnessie/cel/common/types/pb/FieldDescription.java#L396))